view/frontend/web/js/view/payment/adyen-methods.js

```diff
@@ -31,12 +31,12 @@ define(
         cancelCouponAction
     ) {
         'use strict';
-        const paymentMethodComponent = 'Adyen_Payment/js/view/payment/method-renderer/adyen-hpp-method';
+        const paymentMethodComponent = 'Adyen_Payment/js/view/payment/method-renderer/adyen-pm-method';
         const paymentMethodCodes = [
-            'adyen_ideal',
-            'adyen_klarna',
-            'adyen_paypal',
-            'adyen_dotpay',
+            // 'adyen_ideal',
```

Comment:
Should we remove the commented out entries and also add a separate entry for amazonpay since there is a file created for it?